moveit_experimental/jog_arm/config/ur_simulated_config.yaml

```diff
@@ -0,0 +1,36 @@
+use_gazebo: true # Whether the robot is started in a Gazebo simulation environment
```

Comment:
Could you maybe add some more structure (and overview description) to this file?

Reply:
I added a high-level comment and separated out some obviously-related parameters. The remaining parameters don't seem to have any obvious groupings

Reply:
I have proposed some more grouping in a separate commit.